jep/229/README.adoc

```diff
@@ -88,41 +88,34 @@ after a link:https://ci.jenkins.io/job/Plugins/job/log-cli-plugin/job/master/21/
 with a link:https://github.com/jenkinsci/log-cli-plugin/blob/b37970d51e7c2d2d723f39fb7271a263f0d2083d/.mvn/maven.config#L3[custom format]
-and a link:https://github.com/jenkinsci/log-cli-plugin/blob/934670f4a9b84f243f8c0d9bbfc7be30a242e32b/.github/workflows/release.yaml#L37-L46[GitHub Action definition]
-running link:https://github.com/jenkinsci/log-cli-plugin/blob/934670f4a9b84f243f8c0d9bbfc7be30a242e32b/.github/workflows/release.sh#L5[`mvn deploy`]
-with a link:https://github.com/jenkinsci/log-cli-plugin/blob/b37970d51e7c2d2d723f39fb7271a263f0d2083d/.github/workflows/settings.xml#L6-L7[bot token for Artifactory].
+and a link:https://github.com/jenkinsci/log-cli-plugin/blob/b1ee6375b940508709ff64cd8e154ad99eb270ce/.github/workflows/cd.yaml[GitHub Action workflow]
+running link:https://github.com/jglick/jenkins-maven-cd-action/blob/3f1f2fa2739ce0747de9a2545aee5d54917a7abb/run.sh#L4[`mvn deploy`]
+with a link:https://github.com/jglick/jenkins-maven-cd-action/blob/3f1f2fa2739ce0747de9a2545aee5d54917a7abb/settings.xml[bot token for Artifactory].
 
 Note that the deployment skips tests and all other Maven mojos not necessary to produce artifacts, because the GitHub Action first
-link:https://github.com/jenkinsci/log-cli-plugin/blob/934670f4a9b84f243f8c0d9bbfc7be30a242e32b/.github/workflows/release.yaml#L9-L36[verifies that the Jenkins build passed]
+link:https://github.com/jglick/verify-ci-status-action/blob/7ed33273673b4d6d9d8b0533166dfda072b96807/run.sh[verifies that the Jenkins build passed]
 whether the deployment was triggered
-link:https://github.com/jenkinsci/log-cli-plugin/blob/934670f4a9b84f243f8c0d9bbfc7be30a242e32b/.github/workflows/release.yaml#L2-L4[automatically or manually].
+link:https://github.com/jenkinsci/log-cli-plugin/blob/b1ee6375b940508709ff64cd8e154ad99eb270ce/.github/workflows/cd.yaml#L2-L4[automatically or manually].
 It is the responsibility of the
 link:https://github.com/jenkinsci/log-cli-plugin/blob/934670f4a9b84f243f8c0d9bbfc7be30a242e32b/Jenkinsfile#L1[Jenkinsfile]
 to ensure that an appropriate set of tests, static analyses, etc. are run on appropriate platforms.
 
-As of this writing unimplemented:
-
-* Use the Action for Release Drafter to create a release (typically including exactly one entry, from a recently merged PR).
-* Create a new Action wrapping the `release.sh` + `settings.xml` so we do not have this stuff duplicated.
-* Add `block-MRP` profile to `plugin-pom`.
+Release Drafter is configured as an Action as well, but always publishing the draft release under the placeholder name `next` rather than a real version.
+This runs (fairly) quickly after PRs are merged, and just updates a draft release.
+If and when a release is successfully deployed to Artifactory, the deployment script
+link:https://github.com/jglick/jenkins-maven-cd-action/blob/3f1f2fa2739ce0747de9a2545aee5d54917a7abb/run.sh#L5-L8[creates a (lightweight) Git tag and publishes the release].
```

Comment:
is the lightweight bit relevant?

Reply:
Well, somewhat; there is a different GitHub API for creating an annotated tag, which I am not doing.